lib/ruby_lsp/setup_bundler.rb

```diff
@@ -143,6 +147,11 @@ def write_custom_gemfile
         parts << 'gem "debug", require: false, group: :development, platforms: :mri'
       end
 
+      # No `require: false` for ruby-lsp-rails since we need to load the Railtie.
```

Comment:
This shouldn't matter for the custom bundle. The railtie is only loaded if `ruby-lsp-rails` is a part of the main Gemfile.